src/lib/FormGroupLayoutLabel.tsx

```diff
@@ -20,6 +20,7 @@ const FormGroupLayoutLabel = <T extends FieldValues>(props: FormGroupLayoutLabel
   }
 
   if (!label) {
+    // eslint-disable-next-line unicorn/no-null
```

Comment:
as mentioned in the other comment, i'm not too sure about this rule.

Reply:
removed it for the package, hope that is okay? or should i remove it completely?